spec/models/user_spec.rb

```diff
@@ -5,3 +5,9 @@
   it { is_expected.to have_db_column :email }
   it { is_expected.to have_db_column :encrypted_password }
 end
+
+describe 'Factory' do
+  it 'has a valid' do
```

Comment:
Please formulate this message so it makes sense

Reply:
We amended the text to make sense.